src/app_errors/mod.rs

```diff
@@ -201,6 +201,10 @@ pub enum AppErrorKind {
     /// An error occured while hashing a value.
     #[fail(display = "HMAC error: {}", _0)]
     HmacError(String),
+
+    /// An error occured while serializing or deserializing JSON.
```

Comment:
What sort of errors might occur during serialization?

Reply:
Serde says "Serialization can fail if T's implementation of Serialize decides to fail, or if T contains a map with non-string keys." so fair enough...